### ISSUE TITLE
Please add parameter with path to verthash.dat file

### ISSUE THREAD

**Issue by RainbowMiner** [sp-hash/TeamBlackMiner#290]
Thank you for adding the Verthash code - what I am missing is a command-line parameter (like --verhash-data in your spminer). Could you please add this parameter, so that we can reuse an already created/downloaded verthash.dat file?

**Comment by UselessGuru** [comment]
> so that we can reuse an already created/downloaded verthash.dat file?

In https://github.com/Minerx117/NemosMiner/commit/3832eda120ba2bf2094a0448a5f853fc063bd498 I create a sym link to the file:
```
                If ($_.Algorithm -eq "VertHash") { 
                    If ((Get-Item -Path $Variables.VerthashDatPath -ErrorAction Ignore).length -eq 1283457024) { 
                        New-Item -ItemType HardLink -Path ".\Bin\$($Name)\VertHash.dat" -Target $Variables.VerthashDatPath -ErrorAction Ignore | Out-Null
                    }
                    Else { 
                        $PrerequisitePath = $Variables.VerthashDatPath
                        $PrerequisiteURI = "https://github.com/Minerx117/miners/releases/download/Verthash.Dat/VertHash.dat"
                    }
                }
```
Works like a charm.

**Comment by sp-hash** [comment]
yes. implemented in v1.65
